include/deal.II/matrix_free/matrix_free.h

```diff
@@ -640,6 +640,13 @@ class MatrixFree : public Subscriptor
    */
   //@{
   /**
+   * Returns whether a given FiniteElement @p fe is supported by this class.
+   */
+  template <int spacedim>
+  static
+  bool supports_fe (const FiniteElement<dim, spacedim> &fe);
```

Comment:
what about `matrix_free_data.supports(my_fe)` instead? I think it reads better than `matrix_free_data.is_supported(my_fe)`.